### ISSUE TITLE
`npm install` fails

### ISSUE THREAD

**Issue by nhardy** [jsdf/webpack-combine-loaders#6]
v2.0.1 and v2.0.2 fail to install with no error logging, apart from `npm ERR! cb() never called!`.

Tested on Node 6.5.0, 6.9.1.

`yarn install` fails with a more detailed error:

```
Trace: 
  Error: https://registry.yarnpkg.com/webpack-combine-loaders/-/webpack-combine-loaders-2.0.2.tgz: invalid tar file
```

Pinning version to v2.0.0 solves the issue.

**Comment by taikn** [comment]
Having the same issue

**Comment by jsdf** [comment]
Sorry! Looks like yarn publish did something bad. I shoulda tested the published version. I've published 2.0.3 using npm.

**Comment by vyorkin** [comment]
@jsdf @nhardy works for me, thanks
